ros_cross_compile/ros_cross_compile.py

```diff
@@ -23,6 +23,7 @@
 from typing import List
 
 from ros_cross_compile.builders import run_emulated_docker_build
+from ros_cross_compile.docker_client import DockerClient
```

Comment:
Does a global docker_client make sense here?

Reply:
global variables are an anti-pattern

Reply:
singletons are also an anti-pattern, they're just a fancy case of a global variable